pkg/client/kubernetes/base/base.go

```diff
@@ -57,11 +57,11 @@ func NewForConfig(config *rest.Config) (*Client, error) {
 			Jobs:                      {"apis", "batch", "v1"},
 			Namespaces:                {"api", "v1"},
 			PersistentVolumeClaims:    {"api", "v1"},
-			Pods:                   {"api", "v1"},
-			ReplicaSets:            {"apis", "apps", "v1"},
-			ReplicationControllers: {"api", "v1"},
-			Services:               {"api", "v1"},
-			StatefulSets:           {"apis", "apps", "v1"},
+			Pods:                      {"api", "v1"},
```

Comment:
`make verify` results in a dirty git repository:

```
$ git s
On branch pull-226
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   pkg/client/kubernetes/base/base.go
	modified:   pkg/utils/secrets/certificates.go
```

Please fix the formatting.

Reply:
Which version of `go` are you using? 

```
go fmt  pkg/client/kubernetes/base/base.go
```

does nothing.

Reply:
```
$ go version
go version go1.10.2 darwin/amd64
```

Reply:
We agreed to upgrade to 1.11 altogether (I'll raise a separate PR for this).